src/com/TTT/BoardPrinter.java

```diff
@@ -19,17 +19,31 @@ public void printBoard(Board board) {
         int rowSize = 3;
         int finalNewLine = (board.size() - 1);
 
-        for(int spaceIndex = 0; spaceIndex < board.size(); spaceIndex++) {
-            out.print(" " + board.markerAt(spaceIndex)+ " ");
+        for (int spaceIndex = 0; spaceIndex < board.size(); spaceIndex++) {
+            printSpace(board, spaceIndex);
 
-            if((spaceIndex + 1) % rowSize != 0)
+            if ((spaceIndex + 1) % rowSize != 0)
                 out.print("|");
 
-            if(separatorRows.contains(spaceIndex))
+            if (separatorRows.contains(spaceIndex))
                 out.print("\n===========\n");
 
-            if(spaceIndex == finalNewLine)
+            if (spaceIndex == finalNewLine)
                 out.print("\n");
         }
     }
-}
+
+    private void printSpace(Board board, int spaceIndex) {
+        String space = board.markerAt(spaceIndex);
+
+        if (space.equals("X")) {
+            out.print(PlayerMarkers.X.printString());
+        }
+        else if (space.equals("O")) {
+            out.print(PlayerMarkers.O.printString());
+        }
+        else {
+            out.print(" " + space + " ");
+        }
+    }
+}
```

Comment:
nitpick but sometimes you have a new line at the end of your files and sometimes you don't. Prob best to be consistent.